shared/settings/about-container.js

```diff
@@ -1,13 +1,11 @@
 // @flow
+import * as Container from '../util/container'
 import About from './about'
 import {HeaderHoc} from '../common-adapters'
-import {connect} from 'react-redux'
 import {version} from '../constants/platform'
-import {defaultProps, compose} from 'recompose'
 
-import type {Dispatch} from '../constants/types/flux'
-
-const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, navigateAppend}) => ({
+const mapStateToProps = () => ({version})
```

Comment:
Is there a reason not to just put this in `mapDispatchToProps` and leave this omitted?

Reply:
mapDispatchToProps should only be creating action props, not state props, this kind of thing belongs in MSP or mergePRops